### PR TITLE
Section on SBML and Sarah

### PR DESCRIPTION
**Pull request by skeating**

### CODE REVIEW

File: content/People/people.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ subtitle = ""
 [content]
   # Choose which groups/teams of users to display.
   #   Edit `user_groups` in each user's profile to add them to one or more of these groups.
-  user_groups = ["Co-ordinators", "Past Co-odrdinators"]
+  user_groups = ["Co-ordinators", "Past Co-ordinators"]
```
Comment (cjmyers) [inline content/People/people.md:16]: Shouldn't coordinators being one word without a hyphen?

**Review by cjmyers — APPROVED** [review]
I approve the changes with the one comment noted.

File: content/People/people.md (hunk 1 above)
Comment (skeating) [reply content/People/people.md:16]: Ask Jet - these came from her stuff

**Comment by cjmyers** [comment]
@JMante1 could you update co-ordinators to coordinators

**Pull request merged**